src/cli.cpp

```diff
@@ -50,6 +51,10 @@ string Cli::help(const char * progname) {
           longopts[8],
           "Output data in REPORT format (CSV or curses, default curses)",
           "REPORT");
+  txt << mkHelpDoc(
+          longopts[9],
+          "Capture memcached command type (set or get, default get)",
+          "CAPTRUE_TYPE");
```

Comment:
Thanks for your review! Please comment `lgtm` if the PR looks good to you.

```diff
diff --git a/README.md b/README.md
index 798c385..be9a603 100644
--- a/README.md
+++ b/README.md
@@ -18,7 +18,7 @@ Build Status: [![Build Status](https://travis-ci.org/bmatheny/memkeys.png?branch
         -r, --refresh=INTERVAL      Refresh the stats display every INTERVAL ms (default 500)
         -l, --logfile=FILE          Output logs to FILE
         -R, --report=REPORT         Output data in REPORT format (CSV or curses, default curses)
-        -C, --capture=CAPTRUE_TYPE  Capture memcached command type (set or get, default get)
+        -C, --capture=CAPTURE_TYPE  Capture memcached command type (set or get, default get)

         -h, --help                  This help
         -v, --verbose               Increase verbosity. May be used multiple times.
diff --git a/src/cli.cpp b/src/cli.cpp
index 4233d2f..f046707 100644
--- a/src/cli.cpp
+++ b/src/cli.cpp
@@ -54,7 +54,7 @@ string Cli::help(const char * progname) {
   txt << mkHelpDoc(
           longopts[9],
           "Capture memcached command type (set or get, default get)",
-          "CAPTRUE_TYPE");
+          "CAPTURE_TYPE");
   txt << endl;
   txt << mkHelpDoc(longopts[4], "This help", "");
   txt << mkHelpDoc(longopts[5],
```

rebased and pushed --force